app/src/main/java/com/hunguet123/searchImage/ui/favorite/FavoriteViewModel.kt

```diff
@@ -0,0 +1,39 @@
+package com.hunguet123.searchImage.ui.favorite
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.hunguet123.searchImage.base.BaseViewModel
+import com.hunguet123.searchImage.data.PhotoRepository
+import com.hunguet123.searchImage.data.model.Photo
+import com.hunguet123.searchImage.data.repository.PhotoRepositoryImpl
+import kotlinx.coroutines.launch
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+class FavoriteViewModel : BaseViewModel(), KoinComponent {
+    private val photoRepository : PhotoRepository by inject()
+
+    val photosFavorite = MutableLiveData<List<Photo>>()
+
+    fun getPhotosFavorite() {
+        viewModelScope.launch {
+            try {
+                photosFavorite.value = photoRepository.getAllPhotos()
+            } catch (err : Error) {
+                super.error.value = err
```

Comment:
không phải hàm override thì k cần gọi super
```suggestion
               error.value = err
```